core/shared/src/main/scala/zio/ZIO.scala

```diff
@@ -6396,6 +6396,35 @@ object ZIO extends ZIOCompanionPlatformSpecific with ZIOCompanionVersionSpecific
           }
         }
     }
+
+  def render(zio: ZIO.Erased): String =
```

Comment:
Should probably be `private[zio]`
```suggestion
  private[zio] def render(zio: ZIO.Erased): String =
```